nuclear_simulator/data_gen/optimization/optimization_results.py

```
@@ -0,0 +1,269 @@
+"""
+Optimization results data structures.
+
+This module defines the data structures used for storing and analyzing
+initial conditions optimization results.
+"""
+
+import time
+from dataclasses import dataclass, asdict
+from typing import Dict, List, Optional, Any
+from pathlib import Path
```

Ruff (0.11.9)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)